Manual_for_achitecture_browser/index.asciidoc

```diff
@@ -0,0 +1,76 @@
+= Manual for architecture browser
+These content help you to know how you can contribute to the content of the architecture browser .
+
+== Stack specific solution
+A stack specific solution is the place where you can find all the solutions of each stack. 
+You should do each solution on a new branch and not take the master branch because that can lead to conflicts.
+
+== Fork repository
+First of all, you have to fork your own copy of devonfw/architectures to your account, by clicking on the icon "Fork".
+
+image::fork.PNG[Fork_Repository,100,100]
+
+== Open the repository on Gitpod
+The repository is already on your account. You have to click on "code" and copy the url.
+
+image::url.png[URL,300]
+After that, go on your browser and enter "https://gitpod.io#"+url. The repository will be open on the idea Visual Studio.
+
+image::gitpod.png[gitpod,400]
+
+== Create/Change File
+To create a new file, you have to click on the icon "new File".
+
+image::newFile.png[newFile] 
+Then give the name to your file and press "Enter".
+
+To change a file, select this file, change the content and save.
+
+== Upload file
+To upload a file, you have to click on the menu button on the far left
+
+image::menu1.png[menu button] 
+select "File", then choose "open File". Enter the name of your file and click "ok". the selected file will be directly opened in the editor.
+
+== Commit and Push
+First of all, you have to create your own branch, to do it, select the icon "source control".
+
+image::sourceControl.png[source Control]
+
+Go on the icon "view and more action" 
+
+image::menu.png[menu button] 
+
+choose "branch" and select "create branch". Give the name of the branch an press "Enter" .
+
+Now you can already do a commit,for this you must first make sure that a change has been made. Select the icon "source control",go on the icon "view and more action" choose "commit", provide the commit message and press "Enter" to confirm.
+
+You can push, by clicking again on the icon "source control",go on the icon "view and more action" choose "push".
+
+== Create a pull request
+After doing a push, go to your Git account a create a pull request by clicking on "compare & pull request".
+
+image::pr.png[PR]
+
+That is important to accept CLA if not yet done.
+
+== Fetch upstream
```

Comment:
This should be done before you start creating the tutorial and is only relevant when you are building a tutorial that is not your first tutorials.